lib/Dist/Zilla/Plugin/Alien.pm

```diff
@@ -330,6 +357,18 @@ sub _helper_hash {
 	\%helper;
 }
 
+has env => (
+	isa => 'ArrayRef[Str]',
+	is => 'rw',
+	default => sub { [] },
+);
+
+sub _env_hash {
+	my($self) = @_;
+	my %env = map { /^\s*(.*?)\s*=\s*(.*)\s*$/ ? ($1 => $2) : ($_ => '') } @{ $self->env };
```

Comment:
I guess my main contention is allowing the bare syntax below
```ini
# 1. bare syntax
# sets $ENV{BAR} = ''
env = BAR 

# 2. assignment with nothing on right side of equal sign (=)
# also sets $ENV{BAR} = '' as in (1) above
env = BAR =
```

I think we can keep the syntax tight here by only allowing the second method, but that is just what *feels* right to me.

What do you think? I am happy with keeping it as is.